### ISSUE TITLE
[DOC-1322] Models Section take model name instead of default  "{} OBJECT".

### ISSUE THREAD

**Issue by marclave** [scalar/scalar#449]
### Expected Behavior

if a model has a name, we should display it :) 

we can have  "{} OBJECT". as a fallback

### Actual Behavior

 "{} OBJECT". for a model name

### Reproduction steps

```bash
1. on latest upload petstore
2. scroll to models
3. see "{} OBJECT".
```

### Version

latest

### What browsers are you seeing the problem on?

Chrome

<sub>[DOC-1322](https://linear.app/api-docs/issue/DOC-1322/models-section-take-model-name-instead-of-default-object)</sub>

**Comment by hanspagel** [comment]
Unfortunately, I think we don’t have the name available. It’s the name of the reference and when the reference is resolved, the name isn’t part of the data anymore.

But we’ll find a way …

**Comment by marclave** [comment]
fixed in #463
